### ISSUE TITLE
Noise simulations

### ISSUE THREAD

**Issue by thibautlouis** [simonsobs/map_based_simulations#14]
We found (Me and Davide Poletti)  that the noise simulations are a bit difficult to analyse due to the very large scale modes (we are currently checking our end of the pipeline, but the noise power spectra are very very red). A good tool to debug would be to have noise simulation with no power for l<30.  Could you produce that ? (the real maps will probably not have power on these scales anyway due to filtering or lack of convergence of maximum likelihood maps).  Two simulations at nside=4096 per frequency channel would be enough for me.  I will send more details on this later this week (but the sims would be useful in any case).

**Comment by msyriac** [comment]
This is resolved now right? Would be good to check power compared to v3 calculator (Colin's request on #so_ps )

**Comment by zonca** [comment]
maps are ready but haven't been checked yet, see https://github.com/simonsobs/map_based_simulations/pull/15

it would be good to check power

**Comment by msyriac** [comment]
@jcolinhill did a quick check in his FG presentation on today's call -- but need a more detailed one from PS, yes.

**Comment by thibautlouis** [comment]
I have computed the noise power spectra, they now look sensible, however I would need the hit count map at Nside=4096 to make a detailed comparison. @zonca should it be part of the distributed maps? (it would be useful for a lot of people). I have tried to generate it myself using your map sim code, however, I have an issue 'ModuleNotFoundError: No module named 'pysm.units' despite having installed pysm.
